nncf/common/graph/graph.py

```diff
@@ -39,6 +39,9 @@ def node_name(self) -> str:
 
     @property
     def node_type(self) -> str:
+        node_type = self.data.get(NNCFGraph.NODE_TYPE_ATTR)
+        if node_type == 'InputLayer':
+            return NNCFGraphNodeType.INPUT_NODE
```

Comment:
Set up the correct `node_type` attribute during NNCFGraph building. The `node_type` property return value should be identical to what is stored in `self.data`.